src/supertux/world.cpp

```diff
@@ -162,7 +179,11 @@ World::get_basedir() const
 {
   return m_basedir;
 }
-
+dictionary*
+World::get_dictionary()
```

Comment:
If I mark this const, then a const dictionary pointer would be returned, and the dictionaries methods couldn't be called from functions who retrieve the dictionary via this method.